docs/ops/sort/NonMaxSuppression_9.md

```diff
@@ -1,6 +1,6 @@
-# NonMaxSuppression {#openvino_docs_ops_sort_NonMaxSuppression_5}
+# NonMaxSuppression {#openvino_docs_ops_sort_NonMaxSuppression_9}
```

Comment:
please do not delete `NonMaxSupression_5.md` with old documentation and stay it as is and just add new specification file with the correct version

Reply:
@rkazants, done, thanks!